docs/conf/portland/2025/virtual.md

```diff
@@ -0,0 +1,39 @@
+---
+template: {{year}}/generic.html
+---
+
+# Virtual Attendance
+
+We're excited to offer a virtual attendance option for Write the Docs Portland 2025.
+The virtual component will run on **May 5-6** alongside our in-person conference.
+
+## Platform
+
+We'll be hosting the virtual conference on [Venueless](https://venueless.org), 
+a platform that combines video streaming with chat and networking features.
+
+## What's Included
+
+Virtual attendance includes:
+
+* Live streamed talks from the main stage
+* Real-time chat with all attendees
+* Virtual networking spaces  
+* Access to talk recordings after the event
```

Comment:
I'm not sure I'd include this? You also get access to recordings if you have no ticket at all. I wouldn't want to make people think that they have to buy a ticket of some sort to get access.